### ISSUE TITLE
2019-04: Command Line Option --preconditioner-add-well-contributions

### ISSUE THREAD

**Issue by OPMUSER** [OPM/opm-simulators#1844]
Has this command line option 

`
--preconditioner-add-well-contributions     `

been removed?

**Comment by blattms** [comment]
No it is still there and queried.

**Comment by OPMUSER** [comment]
It is missing from` flow -h` output, in 2019-04 release.

**Comment by blattms** [comment]
That's weired. Let me double check in a run.

**Comment by blattms** [comment]
Indeed, we never query this option. Seems like the parameter system
only shows queried ones and ignores the restx?

Somehow this got lost in commit 2d2d0c44336d151df6145 of PR #1618.
@totto82 is there a reason for removing it?

**Comment by totto82** [comment]
> @totto82 is there a reason for removing it?

Probably not.It was some back and forth with this PR, maybe it got lost. I notices even that part of the parameter is still lingering around. 

https://github.com/totto82/opm-simulators/blob/2d2d0c44336d151df614557fa2e973099d3e6ead/opm/autodiff/FlowLinearSolverParameters.hpp#L78

Do we need the feature? It shouldn't be too much effort to re-introduce it again. But if nobody uses the feature, I don't see any point adding this to the code.

**Comment by OPMUSER** [comment]
This is a developer call, from a user point of view I have never used it, so whatever you decide is fine with me.
